packages/devtools/lib/src/main.dart

```diff
@@ -194,6 +195,9 @@ class PerfToolFramework extends Framework {
       disabledTooltip: getDebuggerDisabledTooltip(),
     ));
     addScreen(LoggingScreen());
+    addScreen(SettingsScreen(
+      disabled: !_isFlutterApp,
```

Comment:
nit: make the settings enabled all the time. This page is just as useful for Dart server apps and maybe even when not already connected.

Reply:
Done on my latest revision (ended up just removing disabled flag in the SettingsScreen constructor).
Also, I called the addScreen() method outside the addScreens() since it would require a connected application to be finished.
The navigation works as expected when there's already connected app, but when there's no connected app, it doesn't replace the page

![image](https://user-images.githubusercontent.com/25892469/63052598-eccd5000-bf09-11e9-997b-abcd522bccdb.png)

Any recommendations on how should I handle it when there's no connected app?

Reply:
Hmm. That looks ugly for the no-connected app case. For now, maybe just ensure the flags page doesn't show any output for the non-connected app case and add a TODO that we need to clean this up. In general the way we are showing the Connect dialog when there is no connected app could use some simplification. In addition, it would be nice to be able to change what app you are connected to once devtools is already running which we don't support yet.
Perhaps, in the future on of the settings will just be to connect to a different Dart VM Service.

Reply:
Fixed the behavior for the non-connected app case, it won't do/show anything for now